site/public/js/office-hours-queue.js

```diff
@@ -1,5 +1,5 @@
 /* exported toggleSort, displayStudentHistory */
-/* global buildCourseUrl, displayErrorMessage */
+/* global buildCourseUrl, displayErrorMessage , csrfToken */
```

Comment:
```suggestion
/* global buildCourseUrl, displayErrorMessage, csrfToken */
```

Reply:
Thanks for taking time and giving this PR a look, will continue working on this issue. Just start on this issue, I believe there are several more files need to be modified. Will PR this out when it's officially done. Thanks again :)